cmd2/argparse_completer.py

```diff
@@ -988,7 +988,7 @@ def __init__(self, *args, **kwargs) -> None:
         self._custom_error_message = ''
 
     # Begin cmd2 customization
-    def set_custom_message(self, custom_message: str='') -> None:
+    def set_custom_message(self, custom_message: str = '') -> None:
```

Comment:
I think this aspect of PEP8 predates optional type hinting and shouldn't be applied.  That being said, PyCharm and/or pylint complain, so it is probably good to just do it this way for maintainability.

Reply:
See https://www.python.org/dev/peps/pep-0526/
Look for: `def __init__(self, damage: int, captain: str = None):`

Reply:
Ok, I guess its official then.  I shall attempt to assimilate my thinking to the Borg.